cmd/main.go

```diff
@@ -196,7 +196,17 @@ func main() {
 		Options.Versions, openshiftVersionsMap, Options.ReleaseImageMirror)
 	domainHandler := domains.NewHandler(Options.BMConfig.BaseDNSDomains)
 	eventsHandler := events.New(db, log.WithField("pkg", "events"))
-	hwValidator := hardware.NewValidator(log.WithField("pkg", "validators"), Options.HWValidatorConfig)
+	staticNetworkConfig := staticnetworkconfig.New(log.WithField("pkg", "static_network_config"))
```

Comment:
I know it is not related to this PR but this method looks ugly. Do you think it would be possible to create it in a bit better way?

Reply:
I think `main.go` needs some love and it deserves a separate story for refactoring.

Reply:
@YuviGold What do you think?

Reply:
I agree. Please open a ticket about that.